CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.4)
```

Comment:
Might be useful here to add some comments on what this file is for, i.e. that it is just for building unit tests and not the main model code itself.

Reply:
Can do!